package.json

```diff
@@ -40,7 +40,7 @@
     "react": "16.14.0",
     "react-bootstrap": "^1.0.0-beta.6",
     "react-dom": "^16.14.0",
-    "react-mutation-mapper": "^0.8.12",
+    "react-mutation-mapper": "^0.8.15",
```

Comment:
also update `cbioportal-frontend-commons` and `cbioportal-utils` to the latest versions for consistency?

Reply:
good point